use-cases/school-timetabling/src/main/java/org/acme/schooltimetabling/solver/justifications/RoomConflictJustification.java

```diff
@@ -2,9 +2,13 @@
 
 import ai.timefold.solver.core.api.score.stream.ConstraintJustification;
 
+import org.acme.schooltimetabling.domain.Lesson;
 import org.acme.schooltimetabling.domain.Room;
 
-public record RoomConflictJustification(Room room, long lessonId1, long lessonId2)
+public record RoomConflictJustification(Room room, Lesson lesson1, Lesson lesson2, String description)
         implements ConstraintJustification {
 
+    public RoomConflictJustification(Room room, Lesson lesson1, Lesson lesson2) {
+        this(room, lesson1, lesson2, String.format("Room '%s' is used for lesson '%s' for student group '%s' and lesson '%s' for student group '%s' at '%s %s'", room, lesson1.getSubject(), lesson1.getStudentGroup(), lesson2.getSubject(), lesson2.getStudentGroup(), lesson1.getTimeslot().getDayOfWeek(), lesson1.getTimeslot().getStartTime()));
```

Comment:
As a point of consistency, we use this pattern instead:

```suggestion
        this(room, lesson1, lesson2, "Room '%s' is used for lesson '%s' for student group '%s' and lesson '%s' for student group '%s' at '%s %s'".formatted(room, lesson1.getSubject(), lesson1.getStudentGroup(), lesson2.getSubject(), lesson2.getStudentGroup(), lesson1.getTimeslot().getDayOfWeek(), lesson1.getTimeslot().getStartTime()));
```